src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/domain/faktavurderinger/Vurdering.kt

```diff
@@ -124,15 +130,20 @@ data class VurderingAldersVilkår(
     constructor(
         svar: SvarJaNei,
         vurderingFaktaEtterlevelse: VurderingFaktaEtterlevelseAldersvilkår?,
```

Comment:
Trenger denne å ha `Etterlevelse` i navnet?

Reply:
Smak og behag, @runska ønsket å ha med `etterlevelse` for at det skulle bli ekstra tydelig hvorfor denne dataen lagres